cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -179,9 +182,33 @@ func main() {
 		provisionerOptions = append(provisionerOptions, controller.AdditionalProvisionerNames([]string{supportsMigrationFromInTreePluginName}))
 	}
 
+	var csiNodeInformer cache.SharedIndexInformer
+	if utilfeature.DefaultFeatureGate.Enabled(features.Topology) {
+		// Create informer to prevent hit the API server for all resource request
+		csiNodeInformer := v1beta1.NewCSINodeInformer(clientset, ctrl.ResyncPeriodOfCsiNodeInformer, cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
```

Comment:
I'm not sure indexing by namespace makes sense because CSINode is a non-namespaced object.

Reply:
I'll change to `cache.Indexers{}`